Resources/ServerInfo/_NF/Guidebook/Rules/9_Pirates.xml

```diff
@@ -3,11 +3,19 @@
 
 Pirates must follow their code.
 
+    - [color=#ff0000]The safe zone rules still apply to pirates.[/color]
     - Pirate crews may never exceed a total force of four members, including the Captain, sentient pets and cyborgs.
     -- Round start pets (e.g. Clarrrpy) do not count towards this limit.
     - There may only be one Pirate Captain and one First Mate at any time.
+    - Pirates may not operate without a First Mate or Captain present.
+    -- If the leader of the pirates has been captured or killed, the remaining pirates may make efforts to rescue them.
+    -- First Mates and Captains are required to open their job slot prior to going to cryo, or ahelp if this is not possible. Inform your crew of what they can do in your absence.
     - As a pirate, you must have LOOC permission before kidnapping and holding any civilian hostage.
     -- People kept hostage cannot be forced to do labor under any circumstance.
+    - If a player surrenders, the plundering of their vessel must be kept to a reasonable level.
```

Comment:
"that's reasonable"
"I don't think so"
okay

Reply:
It's the difference between giving and taking some quarter. I think this one has to be left open to some level of interpretation.

Reply:
You can surrender, yes. You are allowed to roleplay being scared in order to save your own skin.

Reply:
So where does 'doing what the antag wants' turn into 'working with the antag'?

Reply:
can a pirate take a hostage or blackmail to force others to fight for them?

Reply:
> So where does 'doing what the antag wants' turn into 'working with the antag'?

I would argue, doing more than they were asked/demanded to, in effort to willingly help

Reply:
> can a pirate take a hostage or blackmail to force others to fight for them?

Hostages can be taken with OOC consent from the potential hostage, and no. Pirates have never been able to recruit from non antags and force them to fight for them under duress. 

> So where does 'doing what the antag wants' turn into 'working with the antag'?
If they tell you to give them some stuff or they'll kill you and you give them some shinies, fine. Especially if you report it afterwards. 

If they ask for things and you immediately just show them to your hidden hold full of C3 because you think it'll benefit them...that's a problem. Especially if you're seen to handle the situation amicably IC. 

Common sense applies a lot in these situations. They're complicated and we will always gather both sides of the story if we need to discuss it with the players involved.

Reply:
I think some clarifications on what is/isn't "helping" the pirate would be nice. Is trading helping? If it is, If you are a food ship, and the pirates pay you for the food, do you need to give it back because they should just steal the food? What if you offer them food instead of explicitly surrendering? Where is the line? I think keeping these things vague is a bad idea, because if anyone stays a kilometer away from anything that could count as "helping" (Trading, surrendering, giving information under pressure) roleplay will become stale and pirates become (or actually, already are) the "Shoot them when you can because they will plunder you and there is nothing you can do" antags, with no room for conversation because the outcome is always the same.

Reply:
It doesn't need to be that complicated. 
Like I said above, if you are being forced to give things to the pirates (emphasis on the word FORCED), then nobody will bwoink you over that. 
However, if you greet them with open arms and immediately show them to your prize hidden stash of C3, then that might raise eyebrows.

Reply:
Additionally, if pirates are paying for food, are they really pirating? 
Pirates don't give, they take.

Reply:
Marking this as resolved. 
People can use their heads. If a player resists, pirates have permission to give no quarter and take what they like. If someone surrenders, a pirate captain can be expected to treat them with some leniency for being a good sport. 
Otherwise it'll get ahelped.